### ISSUE TITLE
[bug] keccak does not work in linux alpine image

### ISSUE THREAD

**Issue by wanseob** [cryptocoinjs/keccak#15]
Tested the tutorial on node:12-alpine docker image shell but failed. Tutorial works well on Ubuntu.

It failed when I try to call `update()` function with "Segmentation fault (core dumped)" error.

**Comment by wanseob** [comment]
Related issues

https://github.com/ethereum/js-ethereum-cryptography#3
https://github.com/ethereumjs/ethereumjs-util/issues/266
https://github.com/ethereumjs/ethereumjs-vm#817

**Comment by alcuadrado** [comment]
@wanseob can you help me reprodce this? Do you have a link to some place where I can do it?

**Comment by PhilippLgh** [comment]
here is a very basic test:
```shell
docker run --rm -it node:alpine /bin/sh
mkdir test
cd test
npm init -y
Wrote to /package.json:

{
  "name": "",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "directories": {
    "lib": "lib",
    "test": "test"
  },
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}


/ # 
/ # npm install --save keccak

> keccak@3.0.0 install /node_modules/keccak
> node-gyp-build || exit 0

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN @1.0.0 No description
npm WARN @1.0.0 No repository field.

+ keccak@3.0.0
added 3 packages from 52 contributors and audited 3 packages in 1.643s
found 0 vulnerabilities

/ # node
Welcome to Node.js v14.5.0.
Type ".help" for more information.
> const createKeccakHash = require('keccak')
> const hash1 = createKeccakHash('keccak256').update('Hello')
Segmentation fault

/ # cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.11.6
PRETTY_NAME="Alpine Linux v3.11"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://bugs.alpinelinux.org/"
```

**Comment by fanatid** [comment]
Thanks for digging! If first sounds easy we can choose this way right now and try a better solution later. Probably I'll able check this issue on weekend.

**Comment by fanatid** [comment]
Published as `3.0.1`.

**Comment by alcuadrado** [comment]
Awesome! Thanks for doing both releases!
